Formula/heirloom-doctools.rb

```diff
@@ -0,0 +1,105 @@
+class HeirloomDoctools < Formula
+  desc "troff, nroff, and related utilities"
+  homepage "https://n-t-roff.github.io/heirloom/doctools.html"
+  url "https://github.com/n-t-roff/heirloom-doctools/releases/download/160308/heirloom-doctools-160308.tar.bz2"
+  sha256 "e4aeae0e5303537755c04226c06d98a46aa35913d1a179708fbc368f93731a26"
+
+  conflicts_with "groff"
+
+  patch :DATA
+
+  def install
+    args = "PREFIX=#{prefix}", "BINDIR=#{bin}", "LIBDIR=#{lib}", "MANDIR=#{man}"
+
+    system "./configure"
+    system "make", *args
+    system "make", "install", *args
+  end
+
+  test do
+    system "#{bin}/troff", "-V"
+  end
+end
+
+__END__
+diff --git a/eqn/eqn.d/Makefile.mk b/eqn/eqn.d/Makefile.mk
+index d6e573c..020c6fe 100644
```

Comment:
@ilovezfs Would PRs like these be easier to review if attached patches were themselves highlighted as diffs? Because I'm happy to arrange that for you. :)